ui/src/app/pages/admin/admin-users.tsx

```diff
@@ -133,17 +133,18 @@ export const AdminUsers = withUserProfile()(class extends React.Component<
   }
 
   convertProfilesToFields(profiles: Profile[]) {
-
+    const usernameWithoutDomain = p => p.username.substring(0, p.username.indexOf('@'));
```

Comment:
Could the `usernameWithoutDomain` utility function be used here?

Reply:
good idea